### PR TITLE
feat: get job status by id

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: Title
Ticket: MEN-6791

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline --fast please

**Comment by merlin-northern** [comment]
@kjaskiewiczz pointed out that I missed the API docs. adding.

**Comment by tranchitella** [comment]
@merlin-northern the ticket referenced in the changelog is wrong.

File: docs/workflows_api.yml [hunk 1]
```diff
@@ -200,6 +200,26 @@ paths:
         409:
           $ref: "#/responses/ConflictError"
 
+  /api/v1/jobs/{id}:
+    get:
+      operationId: Job Structure
+      summary: Gets the job for the given id.
+      description: |
+        Gets the job object for given job id.
```
Comment (tranchitella) [inline docs/workflows_api.yml:208]:
```suggestion
        Gets the job object for the given job id.
```

Comment (merlin-northern) [reply docs/workflows_api.yml]: fixed.

**Comment by merlin-northern** [comment]
> @merlin-northern the ticket referenced in the changelog is wrong.

fixed. thank you.

**Comment by merlin-northern** [comment]
I am not running a full pipeline here, this is not covered in those tests.

**Pull request merged**